nanorl/infra/loop.py

```diff
@@ -54,6 +54,8 @@ def train_loop(
         if i >= warmstart_steps:
             if replay_buffer.is_ready():
                 transitions = replay_buffer.sample()
+                if i <= warmstart_steps + 5:
+                    print(type(transitions.observation))
```

Comment:
confirmed that both jax and torch versions keeps the transitions passed into agent.update as numpy arrays right now